### PR TITLE
Limit displayed rows

### PR DESCRIPTION
**Pull request by LMS235**
Limit the maximum displayed rows by parameter "maxlines"

### CODE REVIEW

File: syntax.php [hunk 1]
```diff
@@ -183,6 +184,11 @@ function render($mode, Doku_Renderer $renderer, $opt) {
             // get next row
             $row = $this->csv_explode_row($content, $opt['delim'], $opt['enclosure'], $opt['escape']);
             $line++;
+            
+            // limit max lines (only if maxlines is not default value 0)
+            if($opt['maxlines'] >= 1 and $opt['maxlines'] == ($line-$opt['hdr_rows'])) {
+				$row = false;
+			}
```
Comment (splitbrain) [inline syntax.php]: indention seems to be messed up here.

**Comment by splitbrain** [comment]
I think introducing a limit also asks for an offset. Could you add that?

And once we have that, pagination might be next.

**Comment by LMS235** [comment]
sorry i'm no programmer :( can you please explain it to me?

**Comment by splitbrain** [comment]
you implemented limit = limiting the result set to a certain number of rows
what's needed is offset = skipping the given number of rows before outputting limit number of rows

**Comment by LMS235** [comment]
add offset, please check

**Comment by splitbrain** [comment]
Indention is still messed up. Can you please check and adjust? Make sure to use 4 spaces, not tabs!

**Comment by LMS235** [comment]
i hope now is ok :)

**Comment by splitbrain** [comment]
Thanks! Could you please update the documentation?

**Comment by LMS235** [comment]
documentation on https://www.dokuwiki.org/plugin:csv is up to date now, thanks for merge

**Pull request merged**